internal/provider/datasources/automation_test.go

```diff
@@ -33,7 +36,7 @@ resource "prefect_automation" "{{ .AutomationResourceName }}" {
         "prefect.resource.id" : "prefect.flow-run.*"
       })
       match_related = jsonencode({
-        "prefect.resource.id" : ["prefect.flow.ce6ec0c9-4b51-483b-a776-43c085b6c4f8"]
+        "prefect.resource.id" : ["prefect.flow.${prefect_flow.test_flow.id}"]
```

Comment:
By adding the `{{ .Deployment }}` resource above, we no longer have to hard-code a reference that only exists in Cloud - this enables us to test in OSS.